### PR TITLE
fix: fix panic when the root is not specified

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

After the OpenDAL was [bumped to v0.27](https://github.com/GreptimeTeam/greptimedb/commit/98ef74bff4507b730369b5a15ba69a31d3206903) , the fs backend builder [will panic if the root is not specified](https://github.com/datafuselabs/opendal/blob/91eebebee1b683659073f9cc751e25293a0fd974/src/services/fs/backend.rs#L147-L153).

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: src/datanode/src/sql/copy_table.rs [hunk 1]
```diff
@@ -52,7 +52,8 @@ impl SqlHandler {
             .context(error::TableScanExecSnafu)?;
         let stream = Box::pin(DfRecordBatchStreamAdapter::new(stream));
 
-        let accessor = Builder::default().build().unwrap();
+        // TODO: Remove hardcode and unwrap
+        let accessor = Builder::default().root("/").build().unwrap();
```
Comment (evenyag) [inline src/datanode/src/sql/copy_table.rs:56]: Would you mind fixing the unwrap here?

**Review by evenyag — APPROVED** [review]
Thanks ❤️ Most LGTM. If you don't mind, could you also help fix the unwrap?

File: src/datanode/src/sql/copy_table.rs (hunk 1 above)
Comment (WenyXu) [reply src/datanode/src/sql/copy_table.rs]: > Would you mind fixing the unwrap here?

I modified it to the following, but it looks ugly... should I define a new error?
```rust
let accessor = Builder::default()
    .root("/")
    .build()
    .context(error::InitBackendSnafu {
        config: Box::new(ObjectStoreConfig::File(FileConfig {
            data_dir: "/".to_string(),
        })),
    })?;
```

**Review by v0y4g3r — APPROVED** [review]
LGTM

**Pull request merged**